compiler-rt/lib/radsan/radsan_user_interface.cpp

```diff
@@ -1,55 +0,0 @@
-/**
-    This file is part of the RealtimeSanitizer (RADSan) project.
-    https://github.com/realtime-sanitizer/radsan
-
-    Copyright 2023 David Trevelyan & Alistair Barker
-    Subject to GNU General Public License (GPL) v3.0
-*/
-
-#include <radsan/radsan.h>
-#include <radsan/radsan_user_interface.h>
-
-#include <sanitizer_common/sanitizer_common.h>
-
-#include <cctype>
-#include <cstring>
-#include <iostream>
-
-namespace radsan {
-
-std::function<OnErrorAction()> createErrorActionGetter() {
-  auto const continue_getter = []() { return OnErrorAction::Continue; };
-  auto const exit_getter = []() { return OnErrorAction::ExitWithFailure; };
-  auto const interactive_getter = []() {
-    auto response = char{};
-
-    std::cout << "Continue? (Y/n): ";
-    std::cin >> std::noskipws >> response;
```

Comment:
This function with std::function, cout, cin, noskipws was the main culprit of problems relating to the C++ standard library